src/main/java/org/spongepowered/common/bridge/data/DataCompoundHolder.java

```diff
@@ -25,9 +25,11 @@
 package org.spongepowered.common.bridge.data;
 
 import net.minecraft.nbt.CompoundNBT;
+import org.spongepowered.api.data.DataHolder;
+import org.spongepowered.common.data.provider.nbt.NBTDataType;
 import org.spongepowered.common.util.Constants;
 
-public interface DataCompoundHolder {
+public interface DataCompoundHolder extends DataHolder {
```

Comment:
I'd prefer bridges not extend their API counterparts, is this required for the compiler?

Reply:
not required but it removed a few unchecked casts